docs/lessons/lesson1-10/Boltdir/site-modules/exercise7/plans/command.pp

```diff
@@ -0,0 +1,3 @@
+plan exercise7::command (TargetSpec $nodes) {
+  run_command("uptime", $nodes)
```

Comment:
Might be nice to return this or add an `out::message` to print it? Though not strictly necessary